### ISSUE TITLE
OversizedPayloadException in VotingServiceSpec

### ISSUE THREAD

**Issue by patriknw** [akka/akka-samples#25]
```
[JVM-1] [ERROR] [05/24/2017 15:16:17.296] [VotingServiceSpec-akka.remote.default-remote-dispatcher-13] [akka.tcp://VotingServiceSpec@localhost:61200/system/endpointManager/reliableEndpointWriter-akka.tcp%3A%2F%2FVotingServiceSpec%40localhost%3A61198-0/endpointWriter] Transient association error (association remains live)
[JVM-1] akka.remote.OversizedPayloadException: Discarding oversized payload sent to Actor[akka.tcp://VotingServiceSpec@localhost:61198/]: max allowed size 128000 bytes, actual size of encoded class akka.actor.ActorSelectionMessage was 139144 bytes
```

also with Akka 2.5.2

**Comment by patriknw** [comment]
To be continued in https://github.com/akka/akka/issues/23025

**Comment by patriknw** [comment]
Fixed in Akka 2.5.3
Refs https://github.com/akka/akka-samples/pull/29
